src/lib/storage/session.js

```diff
@@ -235,6 +238,31 @@ SessionStorage.prototype = {
   },
 
   /**
+   * Get the selected tab index for a group.
+   *
+   * @param {ChromeWindow} chromeWindow
+   * @param {Number} groupID - the groupID
+   */
+  getGroupSelectedIndex: function(chromeWindow, groupID) {
+    let groupsData = this._getGroupsData(chromeWindow);
+    let currentGroup = groupsData[groupID];
+    return currentGroup == null ? 0 : currentGroup.selectedIndex;
+  },
+
+  /**
+   * Set the selected tab index for a group.
+   *
+   * @param {ChromeWindow} chromeWindow
+   * @param {Number} groupID - the groupID
+   * @param {Number} index - the new selected index
+   */
+  setGroupSelectedIndex: function(chromeWindow, groupID, index) {
+    let groupsData = this._getGroupsData(chromeWindow);
+    groupsData[groupID].selectedIndex = index;
```

Comment:
`groupsData` could return an empty object for newly created sessions, so `groupsData[groupID]` is undefined.

@jspenguin Sorry, the comment was meant to be here. I failed at memorizing the line number while switching from the debugger to GitHub...